src/foam/u2/ViewSpec.js

```diff
@@ -111,10 +111,19 @@ foam.CLASS({
         return value;
       }
     ],
-    ['view', { class: 'foam.u2.view.MapView' }],
-    [ 'adapt', function(_, spec, prop) {
-      return foam.String.isInstance(spec) ? { class: spec } : spec ;
-    } ],
+    {
+      name: 'view',
+      value: { class: 'foam.u2.view.MapView' }
+    },
+    {
+      name: 'adapt',
+      value: function(_, spec, prop) {
+        if ( foam.String.isInstance(spec) ) {
+          return spec.startsWith('{') ? JSON.parse(spec) : { class: spec };
```

Comment:
Maybe we should add a spec = spec.trim();

Reply:
Does that affect the JSON.parse?

Reply:
No, but it affects the startsWith().